### PR TITLE
Adding ARM as supported

### PR DESCRIPTION
**Pull request by vmoutoussamy**
Adding ARM as supported

### Description

Describe the overall goals of this pull request.

### Are there any relevant issues/feature requests?

* bsc#...
* jsc#SLE-...

### Is this feature exclusive to SLE 15 SP3 (or Leap 15.3)?

We are currently **only merging documentation relevant for SLE 15 SP2**
(and Leap 15.2) or lower.

- [ ] This PR only applies to SLE 15 SP3 or higher
- [ ] This PR applies to older releases as well.

### Are backports required?

- [ ] To maintenance/SLE15SP2
- [ ] To maintenance/SLE15SP1
- [ ] To maintenance/SLE15SP0
- [ ] To maintenance/SLE12SP5
- [ ] To maintenance/SLE12SP4
- [ ] To maintenance/SLE12SP3
- [ ] To maintenance/SLE12SP2

### CODE REVIEW

File: xml/kvm_virtualization_basics.xml [hunk 1]
```diff
@@ -18,12 +18,12 @@
...
 
   <para os="sles">
-   &kvm; is a full virtualization solution for the &x86-64; and the
+   &kvm; is a full virtualization solution for the &x86-64;, &aarch64; and the
    &zseries; architectures supporting hardware virtualization.
   </para>
 
   <para os="osuse">
-   &kvm; is a full virtualization solution for the &x86-64; architecture
+   &kvm; is a full virtualization solution for the &x86-64;, &aarch64 and the &zseries; architectures
```
Comment (cwickert) [inline xml/kvm_virtualization_basics.xml:26]: The semicolon after &aarch64 is missing. Correct is:
```suggestion
   &kvm; is a full virtualization solution for the &x86-64;, &aarch64; and the &zseries; architectures
```
I would not list z Systems as long as we don't release openSUSE for s390x.

**Review by cwickert — COMMENTED** [review]
Does not build due to a missing semicolon. IHMO we shouldn't mention z System in the openSUSE section as long as it's not released.

**Comment by vmoutoussamy** [comment]
@aginies can we safely list aarch64 as "supported" for KVM with openSUSE?

**Comment by cwickert** [comment]
Assigning this one to myself. I'll take care of merging everything once we have a response from @aginies.

**Comment by aginies** [comment]
Its a Question for the PrjMgr of Arm and the PM for Virtualization. The current statement is false as we do not support all Arm stuff. I don't know the technical details.

**Review by lproven — APPROVED** [review]

**Comment by lproven** [comment]
This phrasing is tricky. 

All modern and recent x86-64 machines, both Intel and AMD, have hardware virtualization. All POWER systems always have, I think, and all z Series.

But not all ARM 64 ones do.

So we need a restrictive subphrase. Saying "Product A supports X, Y and Z supporting A" is not clear at all.

I suggest:

"$PRODUCT supports x86-64, POWER, and z Series, and ARM64 systems which include hardware virtualization."

**Comment by sknorr** [comment]
@tbazant @cwickert Is this still relevant?

**Comment by taroth21** [comment]
@vmoutoussamy: I stumbled across this when cleaning up stale branches. 

The sentence in question here is no longer included (the sentence now reads: "KVM is a full virtualization solution for hardware architectures that support hardware virtualization (refer to Section 7.1, “Architecture support” for more details on supported architectures)." - see also https://susedoc.github.io/doc-sle/main/html/SLES-virtualization/cha-kvm-intro.html#sec-kvm-intro.

I therefore suggest to close this PR unmerged and to remove the branch. @vmoutoussamy: Is this OK with you?

**Comment by vmoutoussamy** [comment]
> @vmoutoussamy: I stumbled across this when cleaning up stale branches.
> 
> The sentence in question here is no longer included (the sentence now reads: "KVM is a full virtualization solution for hardware architectures that support hardware virtualization (refer to Section 7.1, “Architecture support” for more details on supported architectures)." - see also https://susedoc.github.io/doc-sle/main/html/SLES-virtualization/cha-kvm-intro.html#sec-kvm-intro.
> 
> I therefore suggest to close this PR unmerged and to remove the branch. @vmoutoussamy: Is this OK with you?

Yes, I'm fine with this, thank you.

**Pull request closed without merging**